pkg/network/protocols/http/etw_http_service.go

```diff
@@ -121,7 +121,7 @@
 //go:build windows && npm
```

Comment:
Please change the CODEOWNERS so every `etw_*` file in `pkg/network/protocols/http` will be owned by WKIT instead of USM